test/test_cases/run_dry_rising_bubble_2d.jl

```diff
@@ -71,22 +71,38 @@ function run_dry_rising_bubble_2d(
         step!(simulation)
         u = simulation.integrator.u.nhm
 
-        # TODO!: After validation -> set up regression test
         # perform regression check
-        # current_min = -0.0
-        # current_max = 0.0
-        # @test minimum(parent(u.ρw)) ≈ current_min atol = 1e-3
-        # @test maximum(parent(u.ρw)) ≈ current_max atol = 1e-3
-        @test true
+        current_min = -0.0008495977528816157
+        current_max = 348.116343256117
+        @test minimum(parent(u)) ≈ current_min atol = 1e-3
+        @test maximum(parent(u)) ≈ current_max atol = 1e-3
```

Comment:
I would suggest checking one variable (e.g. ρw or ρθ) rather than the entire field, but that's up to you.